Gemfile

```diff
@@ -274,24 +275,24 @@ end
 group :test do
   # RSpec (unit tests, some integration tests)
 
-  gem "fixture_builder",   "0.4.1"
+  gem "fixture_builder",   "0.5.0"
   gem "fuubar",            "2.2.0"
   gem "test_after_commit", "1.1.0"
 
   # Cucumber (integration tests)
 
-  gem "capybara",           "2.12.1"
-  gem "database_cleaner",   "1.5.3"
-  gem "poltergeist",        "1.13.0"
+  gem "capybara",           "2.14.4"
+  gem "database_cleaner",   "1.6.1"
+  gem "poltergeist",        "1.15.0"
 
   gem "cucumber-api-steps", "0.13", require: false
 
   # General helpers
 
   gem "factory_girl_rails", "4.8.0"
-  gem "timecop",            "0.8.1"
-  gem "webmock",            "2.3.2", require: false
-  gem "shoulda-matchers",   "3.1.1"
+  gem "shoulda-matchers",   "3.1.2"
+  gem "timecop",            "0.9.1"
```

Comment:
Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `shoulda-matchers` should appear before `webmock`.